packages/core/src/video/metrics/metrics.ts

```diff
@@ -1,4 +1,4 @@
-export type Metrics = {
+export type RawMetrics = {
```

Comment:
If the rename is not to conflict with the provider `Metrics` type, wdyt of `PlayerMetrics`?